kong/init.lua

```diff
@@ -1504,6 +1504,7 @@ function Kong.log()
   local ctx = ngx.ctx
   if not ctx.KONG_LOG_START then
     ctx.KONG_LOG_START = now() * 1000
+    ctx.KONG_LOG_START_NS = time_ns()
```

Comment:
What about we get the time in ns precision and then round it to 1us?
Also, is there some consideration that we making the `KONG_LOG_START` field with ns precision?

Reply:
> round it to 1us

Would it provide any advantage to do that? Some of the tracing backends do support nanosecond as the smallest precision, others seem to do the rounding up automatically, so I believe we can just keep that level of precision where we can afford it.

The reason I preferred to keep LOG_START and LOG_START_NS separate, rather than setting the value in nanosecond and then converting to ms is that we would end up comparing times that come from different sources when, for example, we use the value `KONG_LOG_START` to compute latencies.

Keeping this value isolated to tracing seemed a better alternative to reduce the impact of the change.